src/io/flutter/logging/FlutterLogFilterPanel.java

```diff
@@ -139,7 +163,13 @@ private void createUIComponents() {
   @NotNull
   private SearchTextField createSearchTextField() {
     final LogFilterTextField logFilterTextField = new LogFilterTextField();
-    logFilterTextField.setOnFilterListener(() -> onFilterListener.onFilter(getCurrentFilterParam()));
+    logFilterTextField.setOnFilterListener(() -> filterListener().onFilter(getCurrentFilterParam()));
     return logFilterTextField;
   }
+
+  @NotNull
+  private OnFilterListener filterListener() {
+    final OnFilterListener listener = onFilterListener;
+    return listener == null ? DEFAULT_ON_FILTER_LISTENER : listener;
```

Comment:
Curious why not just:

    return onFilterListener == null ? DEFAULT_ON_FILTER_LISTENER : onFilterListener;

Reply:
Ah, because of this one
<img width="1149" alt="screen shot 2018-06-27 at 10 11 06 am" src="https://user-images.githubusercontent.com/6185205/41950874-763cd094-79f2-11e8-8f8b-fc0c4af5321c.png">